src/components/KModalFullscreen/KModalFullscreen.vue

```diff
@@ -243,14 +243,14 @@ $fullscreen-modal-padding: 64px;
   width: 100vw;
   z-index: 9999;
 
-  @media only screen and (min-width: ($viewport-md + 1px)) {
+  @media (min-width: ($viewport-md + 1px)) {
```

Comment:
Oh wait these media queries don't look correct. 

You shouldn't need the `+1` when using `min-width` on any of these.

Reply:
I know this is outside the scope of your PR, but do you see what I mean?